README.md

```diff
@@ -86,6 +86,12 @@ the same aspect ratio in KaTeX as in LaTeX.  However, because most browsers
 default to a larger font size, this typically means that a 1cm kern in KaTeX
 will appear larger than 1cm in browser units.
 
+## Libraries
```

Comment:
Thanks for adding this section.  I'm sure we'll be expand this to include other libraries/components.

Reply:
Perhaps [katex-ruby](https://github.com/glebm/katex-ruby) from @glebm would be a good addition.

Reply:
Agreed. Also, wasn't there a KA plugin of some relevance?